app/bundles/CoreBundle/Templating/Twig/Extension/DateExtension.php

```diff
@@ -78,4 +79,12 @@ public function toDate($datetime, $timezone = 'local', $fromFormat = 'Y-m-d H:i:
     {
         return $this->dateHelper->toDate($datetime, $timezone, $fromFormat);
     }
+
+    /**
+     * @see DateHelper::formatRange
+     */
+    public function formatRange($range, $format = null)
```

Comment:
I'd remove the second param as it's unused.
```suggestion
    public function formatRange(\DateInterval $range): string
```
Although by looking at the usage of the formatRange method in the EmailBundle there is a "timestamp" sent to it which is obviously a bug that would be discovered by a proper type. But fixing that is not the point of this PR. Please add it to the method comment like `* @param \DateInterval $range` instead.